src/text.rs

```diff
@@ -112,8 +112,9 @@ impl Rasterizer {
 
             if !glyph_buffer.is_empty() {
                 // Glyph position inside the buffer.
-                let y = anchor_y + ascent - glyph.top as usize;
-                let x = ((anchor_x + offset) as i32 + glyph.left) as usize;
+                let y = usize::try_from((anchor_y + ascent) as i32 - glyph.top).unwrap_or_default();
+                let x = usize::try_from(((anchor_x + offset) as i32 + glyph.left) as isize)
+                    .unwrap_or_default();
```

Comment:
Hm, but negative would be the correct position, right? So I'd assume this would mean that a glyph that would be placed outside of the buffer is instead moved to the side?

It would probably be better to cut off that glyph, right?

Reply:
So you mean pass isizes and cut?

Reply:
Basically, yeah.